HealthyPlanners/Urban-Sherpa
# Favorites front

--NOTE - To the universalForm pull request should be merged in first

**Tasks**
-Implemented Favorites Page by:
-Created Containers for favorite search cards and a title block if needed
-Created Search Card component and added basic styles
-Created Search Card placeholder component and added styles
-Created Favorites redux slice and connected it to store
-Created Favorites NavLink

**To Test**
-"Log In" -- Click the login navLink followed by the login button
-Click the favorites NavLink
-Ten placeholder elements should render to the screen for a brief second followed by three favorites cards

**The Files**
**Components/Routing**
Added favorites route and deleted comments that were causing a linting error. File also renamed to .jsx extension to avoid linting errors.

**CardsContainer**
Renders favorite search cards from state

**Favorites.css**
Basic styles for all components in the favorites folder

**Favorites.jsx**
Renders containers for cards and title block if needed

**SearchCard.jsx**
Renders the favoritesSearch card based on passed in props. 

**AuthenticatedUserDisplay.jsx**
Renamed filed extension to jsx to avoid linting error and added favorites nav link to user authenticated nav bar rendering

**ALL FORM RELATED FILES ARE PART OF ANOTHER PR**

**SearchCardPlaceholder.css & SearchCardPlaceholder.jsx**
Styles and placeholder for searchCards while they load.

**rootReducer.js**
Connected favorites entity to store (and thus react)

**favoritesEntity**
Redux slice for favorites and current search saving

**ALL OTHER FILES ARE PART OF ANOTHER PR**
